commit 55cef04cf9daf5d9765860cd4bb4be1e0118daf5
Author: Nils Andreas Svee <me@lochnair.net>
Date:   Sun Dec 10 01:20:12 2017 +0100

    abuild: add triplet definitions for mips architectures

diff --git a/functions.sh.in b/functions.sh.in
index b886e72..4f2c023 100644
--- a/functions.sh.in
+++ b/functions.sh.in
@@ -9,6 +9,10 @@ arch_to_hostspec() {
 	armel)		echo "armv5-alpine-linux-musleabi" ;;
 	armhf)		echo "armv6-alpine-linux-muslgnueabihf" ;;
 	armv7)		echo "armv7-alpine-linux-musleabihf" ;;
+	mips)           echo "mips-alpine-linux-musl" ;;
+	mips64)         echo "mips64-alpine-linux-musl" ;;
+	mipsel)         echo "mipsel-alpine-linux-musl" ;;
+	mips64el)       echo "mips64el-alpine-linux-musl" ;;
 	ppc)		echo "powerpc-alpine-linux-musl" ;;
 	ppc64)		echo "powerpc64-alpine-linux-musl" ;;
 	ppc64le)	echo "powerpc64le-alpine-linux-musl" ;;
@@ -26,6 +30,10 @@ hostspec_to_arch() {
 	armv6*-*-*-*eabihf)	echo "armhf" ;;
 	armv7*-*-*-*eabihf)	echo "armv7" ;;
 	i[0-9]86-*-*-*)		echo "x86" ;;
+	mips-*-*-*)             echo "mips" ;;
+	mips64-*-*-*)           echo "mips64" ;;
+	mipsel-*-*-*)           echo "mipsel" ;;
+	mips64el-*-*-*)         echo "mips64el" ;;
 	powerpc-*-*-*)		echo "ppc" ;;
 	powerpc64-*-*-*)	echo "ppc64" ;;
 	powerpc64le-*-*-*)	echo "ppc64le" ;;
